src/main/pages/workspace-admin/stacks.adoc

```diff
@@ -8,55 +8,55 @@ folder: workspace-admin
 
-[id="how-stacks-are-used"]
-== How Stacks Are Used?
+See the link:creating-starting-workspaces.html[Creating and starting workspaces user guide] for more information.
 
-As stacks are workspace templates, they are used to create workspace when in User Dashboard. See: link:creating-starting-workspaces.html[User Guide: Creating and Starting Workspaces].
+[id="importing-community-supported-stacks-and-applications"]
+== Importing community supported stacks and applications
 
-[id="community-supported-stacks"]
-== Community Supported Stacks
+Che includes some stacks and sample applications that are pre-configured and tested. Stacks that are contributed by the Che community are not tested. Community stacks are located in the https://github.com/che-samples/community-stacks[community stacks GitHub repository].
 
-The list of stacks Che offers out of the box references tested stacks and sample applications with commands on how to build, run and debug them. There are also stacks contributed by the community which work fine with Che but do not go through daily tests. It is possible however that sample applications may be incompatible with certain stacks.
+Each directory has `${technology}-stack.json` and `${technology}-samples.json`.
 
-Community supported stacks and sample apps (if any) are located in https://github.com/che-samples/community-stacks[Community stacks GitHub repo].
+To import a stack, follow these steps:
 
-Each directory has `${technology}-stack.json` and `${technology}-samples.json`. If you want to import a chosen stack, follow the below instructions:
+.  Copy the content of the JSON.
+.  Go to `${CHE_HOST}/swagger/#!/stack/createStack`.
+.  Paste the JSON content to to request body field.
+.  Click *Try it out* button.
 
-1.  Copy content of a chosen json
-2.  Go to `${CHE_HOST}/swagger/#!/stack/createStack`, paste JSON content to request body field
-3.  Hit *Try it out* button.
+You can choose a different name or ID when there is a conflict with the stack ID or name.
 
-If there’s a conflict with stack ID or name, use a different name or ID (make sure it is unique).
+For a multi-user setup, you can make your stack available for a particular user or all users in the system.  See link:#stack-sharing-and-system-stacks[stack sharing] for more information.
 
-For a multi-user setup, you may want to make your stack available for a particular user or all users in the system. If this is the case, see link:#stack-sharing-and-system-stacks[stack sharing].
+To import sample applications, move `*-stacks.json` files to:
 
-To import *sample applications*, move chosen `*-stacks.json` files to:
+* `${LOCAL_STORAGE}/instance/data/templates` for Docker infrastructure.
+* `${mount-path-of-che-data-volume}/templates` for OpenShift and Kubernetes infrastructure. You need administrator privileges to get the host path and to access the host directory. Also, the new JSONs have the same permissions as the original `samples.json`.
```

Comment:
Should "JSONs" be "JSON files"? Likewise, "`samples.json`" should likely be "`samples.json` file".